### PR TITLE
Update release notes and component list for release-1.6

### PR DESCRIPTION
**Pull request by bhatpmk**
Updates release notes and component list for release 1.6

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -7,13 +7,38 @@ draft: false
 ### v1.6.0
 Features:
 - Added support for Kubernetes v1.25 and v1.26
+- Added a None profile that comes with all components disabled by default
+- Added Thanos, which supports high availability and long-term storage on top of Prometheus
+- Enhanced the Verrazzano CLI to support installing and upgrading from a private registry and in air-gapped environments
+- Added Fluent Operator to allow use of Fluent Bit for log processing, but it will be disabled by default
+- Customer-managed Cert-Manager instances can now be used with Verrazzano
```
Comment (mgianatagh) [inline content/en/docs/releasenotes/_index.md:14]: Add something like this:

Added support to create OCNE clusters from the Rancher UI

File: content/en/docs/setup/install/verify/softwares.md [hunk 2]
```diff
@@ -9,31 +9,34 @@ Verrazzano installs a curated set of open source software. The following table l
...
+  | [Argo CD]({{< relref "/docs/reference/vpo-verrazzano-v1beta1#install.verrazzano.io/v1beta1.ArgoCDComponent" >}})                                | 2.7.2        | A declarative, GitOps continuous delivery tool for Kubernetes.                           |
   | [cert-manager]({{< relref "/docs/reference/vpo-verrazzano-v1beta1#install.verrazzano.io/v1beta1.CertManagerComponent" >}})                      | 1.9.1        | Automates the management and issuance of TLS certificates.                               |
-  | [Coherence Operator]({{< relref "/docs/reference/vpo-verrazzano-v1beta1#install.verrazzano.io/v1beta1.CoherenceOperatorComponent" >}})          | 3.2.9        | Assists with deploying and managing Coherence clusters.                                  |
+  | [cert-manager-webhook-oci]({{< relref "/docs/reference/vpo-verrazzano-v1beta1#install.verrazzano.io/v1beta1.CertManagerWebhookOCIComponent" >}})| 0.1.0        | ACME webhook for Oracle Cloud Infrastructure.                                            |
```
Comment (michael-cico) [inline content/en/docs/setup/install/verify/softwares.md:14]: Same here for the webhook

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (bhatpmk) [reply content/en/docs/releasenotes/_index.md:14]: Done

File: content/en/docs/setup/install/verify/softwares.md (hunk 2 above)
Comment (bhatpmk) [reply content/en/docs/setup/install/verify/softwares.md]: Done

**Review by michael-cico — APPROVED** [review]

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -12,28 +12,44 @@ draft: false
...
 - WebLogic Monitoring Exporter v2.1.3
+- NGINX Ingress Controller v1.7.1
+- Prometheus Operator v0.64.1
+- Prometheus v2.44.0
+- kube-state-metrics v2.8.2
+
+#### Components added:
+- Thanos v0.30.2
+- Fluent Operator v2.2.0
+- FluentbitOpensearchOutput Helm chart v1.6.0
```
Comment (kminder) [inline content/en/docs/releasenotes/_index.md:37]: Will this mean anything to anyone outside the Verrazzano team?  If we keep it should the "Helm chart" be dropped?

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -12,28 +12,45 @@ draft: false
...
-- Added support for Kubernetes v1.25 and v1.26
+- Added support for Kubernetes v1.25 and v1.26.
+- Added support to create OCNE clusters from the Rancher UI.
+- Added Thanos, which supports high availability and long-term storage on top of Prometheus.
+- Added Fluent Operator to allow use of Fluent Bit for log processing, but it will be disabled by default.
+- Enhanced the Verrazzano CLI to support installing and upgrading from a private registry and in air-gapped environments.
+- Customer-managed Cert-Manager instances can now be used with Verrazzano.
+- Added a None profile that comes with all components disabled by default.
 
 #### Component version updates:
```
Comment (fredtibbitts) [inline content/en/docs/releasenotes/_index.md:23]: ArgoCD was upgraded so it should be added to this list.

**Review by rosemarymarano — COMMENTED** [review]
Edits. PS Don't forget to change the 1.6 release date in the Prerequisites doc.

**Comment by bhatpmk** [comment]
> Edits. PS Don't forget to change the 1.6 release date in the Prerequisites doc.

I think we will do that once release-1.6 branch is there. We need to edit some version change in config.toml as well

File: content/en/docs/releasenotes/_index.md [hunk 5]
```diff
@@ -12,28 +12,45 @@ draft: false
 - Enhancement_n
 
 #### Features:
-- Added support for Kubernetes v1.25 and v1.26
+- Added support for Kubernetes v1.25 and v1.26.
+- Added support to create OCNE clusters from the Rancher UI.
+- Added Thanos, which supports high availability and long-term storage on top of Prometheus.
+- Added Fluent Operator to allow use of Fluent Bit for log processing, but it will be disabled by default.
+- Enhanced the Verrazzano CLI to support installing and upgrading from a private registry and in air-gapped environments.
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/releasenotes/_index.md]: Done

File: content/en/docs/releasenotes/_index.md [hunk 6]
```diff
@@ -12,28 +12,44 @@ draft: false
 - Enhancement_n
 
 #### Features:
-- Added support for Kubernetes v1.25 and v1.26
+- Added support for Kubernetes v1.25 and v1.26.
+- Added support to create OCNE clusters from the Rancher UI.
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/releasenotes/_index.md]: Done

File: content/en/docs/releasenotes/_index.md [hunk 7]
```diff
@@ -12,28 +12,44 @@ draft: false
...
+- Added Thanos, which supports high availability and long-term storage on top of Prometheus.
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/releasenotes/_index.md]: Done

File: content/en/docs/releasenotes/_index.md [hunk 8]
```diff
@@ -12,28 +12,45 @@ draft: false
...
+- Added Fluent Operator to allow use of Fluent Bit for log processing, but it will be disabled by default.
+- Enhanced the Verrazzano CLI to support installing and upgrading from a private registry and in air-gapped environments.
+- Customer-managed Cert-Manager instances can now be used with Verrazzano.
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/releasenotes/_index.md]: Done

File: content/en/docs/releasenotes/_index.md [hunk 9]
```diff
@@ -12,28 +12,45 @@ draft: false
...
+- Prometheus Operator v0.64.1
+- Prometheus v2.44.0
+- kube-state-metrics v2.8.2
+- Coherence Operator v3.2.10
+
+#### Components added:
+- Thanos v0.30.2
+- Fluent Operator v2.2.0
+- FluentbitOpensearchOutput Helm chart v1.6.0
+- Cert-Manager webhook for Oracle Cloud Infrastructure DNS v0.1.0
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/releasenotes/_index.md]: Done

File: content/en/docs/releasenotes/_index.md [hunk 10]
```diff
@@ -12,28 +12,45 @@ draft: false
...
-- Fix_2
-- ...
-- Fix_n
+- Disabled the startupapicheck job in cert-manager startup.
+- Fixed an issue with Rancher SSO via Keycloak when using a custom CA for Verrazzano certificates.
+- Fixed an issue where not defining the replicas for default OpenSearch node pools in the Verrazzano CR caused the pods to terminate.
+- Updated base image and other component images to resolve CVEs.
 
 #### Known issues and workarounds:
 - Systemd journal logs are not collected on some platforms. The [systemd plugin for Fluentd](https://github.com/fluent-plugin-systemd/fluent-plugin-systemd) cannot read systemd journal files on Kubernetes nodes that use newer versions of systemd. This is known to impact the default node images of [kind](https://kind.sigs.k8s.io/) for Kubernetes 1.24 and later, Oracle Linux 9, and any other Kubernetes cluster nodes running on an operating system that has systemd version 246 or higher.**
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/releasenotes/_index.md:50]: This was done by an earlier commit https://github.com/verrazzano/docs/commit/f108373e84ea77f3f96a5ad628a405ae1212d891

[the remark this answers is not in the document]
Comment (rosemarymarano) [reply content/en/docs/releasenotes/_index.md:50]: @bhatpmk Please change it to: 
-> that has systemd version 246 or later. (no asterisks)

**Review by rosemarymarano — COMMENTED** [review]
One more minor fix and the fix for the Known Issue text.

File: content/en/docs/releasenotes/_index.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/releasenotes/_index.md]: Done

File: content/en/docs/releasenotes/_index.md [hunk 11]
```diff
@@ -12,28 +12,45 @@ draft: false
...
 
 #### Features:
-- Added support for Kubernetes v1.25 and v1.26
+- Added support for Kubernetes v1.25 and v1.26.
+- Added support to create OCNE clusters from the Rancher UI.
+- Added Thanos, which supports high availability and long-term storage on top of Prometheus.
+- Added Fluent Operator to allow use of Fluent Bit for log processing, but it will be disabled by default.
+- Enhanced the Verrazzano CLI to support installing and upgrading from a private registry and in air-gapped environments.
+- Customer-managed Cert-Manager instances can now be used with Verrazzano.
+- Added a None profile that comes with all components disabled by default.
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/releasenotes/_index.md]: I have removed none profile from the release notes. I will add it back, if it is required

File: content/en/docs/releasenotes/_index.md (hunk 3 above)
Comment (bhatpmk) [reply content/en/docs/releasenotes/_index.md]: @kminder After consulting @adiforluls , I have removed this entry from release notes.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**